src/book/spectra/pxrd/index.yml

```diff
@@ -4,3 +4,6 @@ tips:
   - minRev: 95
     title: Simulation from CIF file
     name: simulation
+  - minRev: 95
```

Comment:
When adding TIPs we can specifiy the minimal version in which this feature was added. Some people work on specific revision and we don't want to display a feature to which they don't have access yet.
The revision can be found in
![image](https://user-images.githubusercontent.com/1484241/107478502-db561a00-6b79-11eb-8af1-42f47786906e.png)
Current revision is 290.

Reply:
I will take care to change it